libutils/logging.c

```diff
@@ -223,6 +224,11 @@ static void LogToConsole(const char *msg, LogLevel level, bool color)
     }
 
     fflush(stdout);
+
+    if (level == LOG_LEVEL_DEBUG)
+    {
+        fsync(STDOUT_FILENO);
```

Comment:
@vpodzime @olehermanse well... wattya know... this fsync(STDOUT_FILENO) is I think what is breaking the serverd-multi-versions.sh tests. I imagine it is because of threading in cf-serverd and getting a deadlock? Not sure of the root cause but definitely see the improvement when I remove fsync().

Reply:
It should not be a problem, but probably not worth wasting time on this now. After all, it has been added as a sort of a "best-effort" thing anyway, right?